apaq-v1.0.yaml

```diff
@@ -0,0 +1,57 @@
+openapi: 3.0.0
+
+info:
+  title: Airport Air Quality Prediction API
+  version: 1.0.0
+  description: API for predicting air quality based on airport information
+
+basePath: /v1
+
+paths:
+  /predict-air-quality:
```

Comment:
I don't really like the name of this path... I think it'd better to go with something like `air-quality`